tutorials/Tutorial9_DPR_training.ipynb

```diff
@@ -0,0 +1,637 @@
+{
```

Comment:
<p>If I remember correctly our implementation actually does not require "negative_ctxs" as we only use in-batch negatives plus the hard negatives. Can you please verify?</p><p>Maybe explain briefly what these different ctxs mean and how to get them...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack/pull/708/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>